docker-compose.override-template.yml

```diff
@@ -0,0 +1,25 @@
+version: "3"
```

Comment:
move this into templates folder `docs/templates/docker-compose.override-template.yml`